src/components/studio-footer/StudioFooter.jsx

```diff
@@ -139,8 +147,10 @@ const StudioFooter = ({
 };
 
 StudioFooter.propTypes = {
-  // injected
-  intl: intlShape.isRequired,
+  containerProps: Container.propTypes,
+};
+
+StudioFooter.defaultProps = {
```

Comment:
`containerProps` should be optional, and/or populated with a default value, correct?

Reply:
Fixed here: 41d76ccda54c89b144f13896e8835d2141d4483d